test/files/neg/t8700.check

```diff
@@ -0,0 +1,7 @@
+Bar.scala:2: warning: match may not be exhaustive.
```

Comment:
That looks like the right line number to me.

```
⚡ scalac sandbox/test.scala
sandbox/test.scala:2: warning: match may not be exhaustive.
It would fail on the following input: List(_)
    def baz(foo: List[String]) = foo match {
                                 ^
one warning found
```